internal/command/checks/handlers.go

```diff
@@ -0,0 +1,162 @@
+package checks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/helpers"
+	"github.com/superfly/flyctl/internal/config"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/flyctl/internal/render"
+	"github.com/superfly/flyctl/iostreams"
+)
+
+func runListChecksHandlers(ctx context.Context) error {
+	out := iostreams.FromContext(ctx).Out
+	web := client.FromContext(ctx).API()
+	org := flag.FirstArg(ctx)
+
+	handlers, err := web.GetHealthCheckHandlers(ctx, org)
+	if err != nil {
+		return err
+	}
+
+	if config.FromContext(ctx).JSONOutput {
+		return render.JSON(out, handlers)
+	}
+
+	fmt.Fprintf(out, "Health Check Handlers for %s\n", org)
+	table := helpers.MakeSimpleTable(out, []string{"Name", "Type"})
+	for _, handler := range handlers {
+		table.Append([]string{handler.Name, handler.Type})
+	}
+	table.Render()
+	return nil
+}
+
+type createHandlerFn func(context.Context, *api.Organization, string) error
+
+func runCreateChecksHandler(ctx context.Context) error {
+	web := client.FromContext(ctx).API()
+	orgSlug := flag.GetString(ctx, "organization")
+
+	handlerFn := map[string]createHandlerFn{
+		"slack":     setSlackChecksHandler,
+		"pagerduty": setPagerDutyChecksHandler,
+	}
+
+	handlerType := flag.GetString(ctx, "type")
+	fn, ok := handlerFn[handlerType]
+	if !ok {
+		return fmt.Errorf("\"%s\" is not a valid handler type", handlerType)
+	}
+
+	org, err := selectOrganization(ctx, web, orgSlug)
+	if err != nil {
+		return err
+	}
+
+	name := flag.GetString(ctx, "name")
+	if name == "" {
+		prompt := &survey.Input{Message: "Name:"}
```

Comment:
Package prompt makes this less verbose

Reply:
thanks for pointing to it. I replaced it all in 2baef524